components/style/properties/shorthand/position.mako.rs

```diff
@@ -517,6 +552,19 @@
             }
 
             if self.grid_auto_flow.autoflow == AutoFlow::Column {
+                // It should fail to parse if other branch's values are set.
```

Comment:
Oops, it should be serialize. By other branch, I meant other branch of the if condition.